application.py

```diff
@@ -92,18 +92,18 @@ async def redoc_html():
 
 @data_service_app.exception_handler(EmptyResultSetException)
 async def empty_result_set_exception_handler(
-    request,  # pylint: disable=unused-argument
-    exc):
-    logger.exception(exc)
+        request,  # pylint: disable=unused-argument
```

Comment:
The indent should be 4 spaces, not 8

Reply:
It is 4 spaces pr. level, request is at level 2, so 8 spaces seems correct. I don't quite see the problem (?)
Identation was done by Intellij, by the way.

Reply:
Most python linters I see use this formatting:
```py
def my_function(
    some, arguments, here 
):
    #function code
```
Here the arguments are indented by 4 spaces as you would suspect and the closing parenthesis is at the same level
as the "def" (0 spaces). Then the code for the function is indented by 4 spaces.


IntelliJ seems to by default prefer this:
```py
def my_function(
        some, arguments, here):
    #function code
```
Since it closes the parenthesis on the indented line, it prefers to indent by 8 to avoid the arguments from being indented at the same level as the function code. I personally don't prefer this as it results in a double indented line that I find off putting.

Reply:
After code reformatting, my IntelliJ uses 4 spaces:
```
async def empty_result_set_exception_handler(
    request,  # pylint: disable=unused-argument
    exc):
    logger.exception(exc)
```
Also with arguments on one line:
```
async def not_found_exception_handler(
    request, exc):
    logger.exception(exc)
```
I use standard settings (Settings -> Editor -> Code style -> Python -> Scheme: Default).

Reply:
This will make the indented arguments the same indentation level as the next line, which is discouraged:

```py
# Further indentation required as indentation is not distinguishable.
def long_function_name(
    var_one, var_two, var_three,
    var_four):
    print(var_one)
```

The double-level indent from the code in this PR is technically OK according to the style guide (My linter would not complain about it).
For me it's just personal preference to use the single indent plus closing parenthesis on the next line. (Black code style)

Reply:
Yes, same indentation level on both arguments and the code of method is not that readable 👍